capture_test.go

```diff
@@ -9,53 +9,55 @@ import (
 )
 
 //goland:noinspection GoNilness
-func TestCaptureContext(t *testing.T) {
+func TestCapture(t *testing.T) {
 	type args struct {
-		ctx     context.Context
-		env     []string
-		command string
-		args    []string
+		ctx  context.Context
+		dir  string
+		env  []string
+		name string
+		args []string
 	}
 	tests := []struct {
 		name    string
 		args    args
-		want    command.Result
+		want    command.Capture
 		wantErr bool
 	}{
 		{
-			name: "invalid command",
+			name: "invalid name",
 			args: args{
-				ctx:     context.Background(),
-				command: "asdfasdf",
+				ctx:  context.Background(),
+				name: "asdfasdf",
 			},
 			wantErr: true,
-			want: command.Result{
+			want: command.Capture{
 				Output:   "",
 				ExitCode: 0,
 			},
 		},
 		{
 			name: "success return",
 			args: args{
-				ctx:     context.Background(),
-				command: "/bin/bash",
-				args:    []string{"-c", "exit 0"},
+				ctx:  context.Background(),
+				dir:  ".",
```

Comment:
what was the reason for using explicit "." vs leaving the default behavior?

Reply:
to make sure it's copied to the object.